package.json

```diff
@@ -6,25 +6,19 @@
   },
   "packageManager": "pnpm@7.14.1",
   "scripts": {
-    "lerna": "lerna",
-    "start": "pnpm watch",
-    "dev": "pnpm start",
-    "test": "pnpm --filter \"@trpc/server\" exec jest",
+    "build": "turbo --filter=\"@trpc/*\" build",
+    "build:www": "turbo --filter www build",
+    "dev": "turbo --filter=\"@trpc/*\" dev",
+    "dev:www": "turbo --filter www dev",
+    "test": "turbo --filter tests test",
```

Comment:
How do we run tests in watch mode? 👀

Reply:
I added a `test-watch` but there's a really big delay when editing a file... before it was using the uncompiled versions. Do you know how we could work around this? I read https://turbo.build/repo/docs/handbook/testing but it doesn't talk about using uncompiled versions

Reply:
That's interesting... Hadn't tested that one.

Not sure what's different from previous. We didn't have a test-watch script, what did you use?

Reply:
```json
"test-watch": "pnpm --filter tests exec jest --watch",
```

Seem to be a bit faster? Still the typechecking makes it quite sluggish still...